### PR TITLE
Module 1: Hello World Project

### PR DESCRIPTION
**Pull request by seyioyemade**
## Module 1: Hello World Project

In this project I:

- Created a branch according to the [GitHub flow].
- Added a .gitignore file.
- Added node_modules to gitignore file.
- Set up a linter for HTML and CSS.
- Added descriptive README file.
- Added a .html file that includes a header with the text "Hello Microverse!". 
- Added a .css file that includes one class with styles for the header.
- Added a test.md file to gitignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# hello-microverse
\ No newline at end of file
+# hello-world
+
+Hello World's goal is to master all of the tools and best practices of web development.
+
+# Built With
+
+HTML and CSS
```
Comment (osamaashraf6) [inline README.md]: - [ ] Kindly 😊 use [the Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @seyioyemade 😊, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

you-are-near


## Highlights 💯🎉

- [x] Good descriptive pull request
- [x] Good following GitHub flow 


## Required Changes ♻️

- [ ]  Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, for example: "Add linters configuration" instead of "add linters configuration" ... etc 👍
- [ ] You did a great job 😃 at adding the configuration for the linter, but as you see here there is an error linter at the eslint, so please 😊 try to fix that to remove the error of the linter I suggest you use the ```  npx eslint . --fix ``` command in the terminal to fix the error of linter at js 💯


- Check the comments under the review._

## Optional suggestions
 N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @seyioyemade, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- commit messages ✔️ 
- Linters passed ✔️

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] I see your readme is yet to use the professionally provided template by Microverse, as suggested by the previous reviewer, I would implore you to use it and adapt it to your own project by filling it out according to your project so that you or any other visitor can easily grasp what the repo entails at a glance. You can find the template [here](https://github.com/microverseinc/readme-template)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by seyioyemade** [comment]
Hi Olaoluwa,

Thank you for reviewing my project.

I'm left with just one request review and I made sure I used the Microverse
readme template.

I don't know what to do from here. Kindly elaborate on your review.

Please help me.

Seyi Oyemade

**Comment by seyioyemade** [comment]
@Olamarx

Thank you for reviewing my project.

I'm left with just one request review and I made sure I used the Microverse readme template.

I don't know what to do from here. Kindly elaborate on your review.


Please help me.

Seyi Oyemade

**Comment by Olamarx** [comment]
I think you removed majority of the content then, kindly try not to remove, just replace with your own content. Do you understand?

**Comment by seyioyemade** [comment]
Okay. 
Thank you.

**Comment by seyioyemade** [comment]
@Olamarx 

Thank you for your time earlier.

I have redone and added the Microverse Readme template properly.

Kindly help check the readme file in my last commit to ensure I did it correctly this time. I have one review chance left in my dashboard  🙏

**Comment by seyioyemade** [comment]
Thank you for your time earlier.

I have redone and added the Microverse Readme template properly.

Kindly help check the readme file in my last commit to ensure I did it
correctly this time. I have one review chance left on my dashboard 🙏


Please.

File: README.md [hunk 2]
```diff
@@ -1 +1,185 @@
...
+Install this project with:
+
+-Copy the URL [https://github.com/seyioyemade/hello-world.git].
+
+-In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+git clone [https://github.com/seyioyemade/hello-world.git]
+
```
Comment (souad988) [inline README.md:101]: [ OPTIONAL ]
well done @seyioyemade, you implemented all this project requirements and applied all changes required by previous reviewers 💪, one small thing to mention is that the content of this section should move to setup and here all user have to do is run ' npm install '.

**Review by souad988 — APPROVED** [review]
Hi @seyioyemade , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔Good PR description 
✔Good commit messages.
 ✔Clear and good communication with reviewers in the comments.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by seyioyemade** [comment]
Thanks for the feedback! 😊

**Pull request merged**